ftprime/argrecorder.py

```diff
@@ -35,6 +36,8 @@ def add_individual(self, name, time, population=msprime.NULL_POPULATION,
             self[name] = (msprime.Node(time=time, population=population,
                                        name=name, is_sample=is_sample), [])
             self.num_nodes = max(self.num_nodes, 1+int(name))
+        else:
+            raise ValueError("Attempted to add " + str(name) + " as a new individual, who wasn't.")
```

Comment:
somewhat confusing. equivalent to? `Attempted to add " + str(name) + ", who already exists, as a new individual."`